package.json

```diff
@@ -80,8 +80,7 @@
     }
   },
   "lint-staged": {
-    "./src/*.{ts,tsx}": [
-      "prettier --write",
```

Comment:
Are we removing prettier?

Reply:
Apparently prettier cannot find any matching files, which prevent us from committing. Very odd problem, need to look into this, will create an issue for this

Reply:
Ok, let's deal witht that later